packages/jest-snapshot/src/__tests__/matcher.test.ts

```diff
@@ -5,7 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import {type Context, toMatchSnapshot} from '../';
+import {Context, toMatchSnapshot} from '../';
```

Comment:
Hm.. This change looks unnecessary or do I miss something?

Reply:
My IDE told me it is `Unused import specifier type` and removed it automatically. I will bring it back.